package.json

```diff
@@ -96,39 +96,39 @@
     "recompact": "^2.2.2",
     "redis": "^2.6.5",
     "redux": "^3.6.0",
-    "redux-logger": "^2.8.1",
-    "redux-observable": "^0.13.0",
+    "redux-logger": "^2.8.2",
+    "redux-observable": "^0.14.0",
     "rxjs": "^5.2.0",
     "sinon": "^1.17.7",
     "tmp": "^0.0.31",
     "uuid": "^3.0.1",
     "webpack": "^2.2.1",
-    "whatwg-fetch": "^2.0.2"
+    "whatwg-fetch": "^2.0.3"
   },
   "devDependencies": {
     "argos-cli": "^0.0.2",
     "babel-eslint": "^7.1.1",
     "clear-require": "^2.0.0",
-    "eslint": "^3.16.1",
+    "eslint": "^3.17.0",
     "eslint-config-airbnb": "^14.1.0",
     "eslint-find-rules": "^1.14.3",
-    "eslint-plugin-babel": "^4.0.1",
+    "eslint-plugin-babel": "^4.1.0",
     "eslint-plugin-import": "^2.1.0",
     "eslint-plugin-jsx-a11y": "^3.0.2",
     "eslint-plugin-mocha": "^4.7.0",
     "eslint-plugin-react": "^6.10.0",
     "factory-girl": "^4.2.1",
     "foreman": "^2.0.0",
-    "jest": "^18.1.0",
+    "jest": "^19.0.2",
```

Comment:
I found an issue relative to notify, so I disabled it. I think if tests are passing, it's safe.